app/assets/javascripts/controllers/ems_common/arbitration_profile_form_controller.js

```diff
@@ -35,6 +35,9 @@ ManageIQ.angular.app.controller('arbitrationProfileFormController', ['$scope', '
         $scope.arbitrationProfileModel.security_group_id    = convertToString(arbitrationProfileData.security_group_id);
 
         $scope.profileOptions($scope.arbitrationProfileModel.ems_id, $scope.arbitrationProfileModel.cloud_network_id);
+
+        $scope._authentication = _.find($scope.arbitrationProfileModel.authentications, {id: $scope.arbitrationProfileModel.authentication_id})
```

Comment:
Oh, never mind :)

It's that `convertToString` .. causes those two to be of different types :)